src/lunabot_bringup/lunabot_bringup/mission_evidence.py

```
@@ -0,0 +1,406 @@
+            os.killpg(process.pid, signal.SIGKILL)
+            return int(process.wait(timeout=5.0))
+
+
+def run_recording(plan: EvidencePlan) -> tuple[int | None, int]:
+    """Run rosbag recording, optionally around a mission command."""
+    record_log = (plan.logs_dir / "rosbag_record.log").open(
+        "w",
+        encoding="utf-8",
+    )
+    try:
+        recorder = subprocess.Popen(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)